src/main/java/seedu/duke/model/itemlist/BookList.java

```diff
@@ -43,10 +43,13 @@ public void addBook(Book book) {
 
     @Override
     public void addTodo(String description) {
-        Book newBook = new Book(description);
-        items.add(newBook);
-        Ui.dukePrint(Messages.MESSAGE_ADD_TASK + newBook.toString() + Messages.MESSAGE_STATUS_FIRST
-                + items.size() + Messages.MESSAGE_STATUS_LAST);
+    }
+
+    /**
+     * Clears all the books in the list.
+     */
+    public void clearBook() {
+        items = new ArrayList<>();
```

Comment:
Is it possible that a single method is created in ItemList which is called clear() so we don't have to duplicate this code in every ItemList subclass?